src/components/StrategiesListTable/StrategiesList.columns.js

```diff
@@ -51,7 +51,7 @@ const activeStrategiesColumns = (t, getMarketPair) => [
     headerStyle: STYLES.flexStart,
     width: 100,
     flexGrow: 1,
-    cellRenderer: ({ rowData = {} }) => defaultCellRenderer(resultNumber(rowData.results?.pf)),
```

Comment:
lets remove defaultCellRenderer when resultNumber is used, as defaultCellRenderer does the same truncate+toolip